noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1648,40 +1898,15 @@
                 ]]>
             </errors>
         </queries>
-    </rule>
-
-    <rule>
-        <title>Sign Off count</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT r._ref_arkivdel series_system_id, COUNT(*) number_of_sign_offs
-                    FROM arkivstruktur.registrering r
-                    WHERE r._dtype = 'journalpost' AND r.systemid in (SELECT _ref_registrering FROM arkivstruktur.avskrivning WHERE _ref_registrering IS NOT NULL GROUP BY _ref_registrering)
-                    GROUP BY r._ref_arkivdel;
-                ]]>
-            </info>
-        </queries>
-    </rule>
-
-    <rule>
-        <title>Document Flow count</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT r._ref_arkivdel series_system_id, COUNT(*) number_of_document_flows
-                    FROM arkivstruktur.registrering r
-                    WHERE r._dtype = 'journalpost' AND r.systemid in (SELECT _ref_registrering FROM arkivstruktur.dokumentflyt WHERE _ref_registrering IS NOT NULL GROUP BY _ref_registrering)
-                    GROUP BY r._ref_arkivdel;
-                ]]>
-            </info>
-        </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST19">
         <title>Screening count</title>
+        <description>
+            Provides information about the number of screened Series, Classes,
```

Comment:
`Provides information about the number of screened series, classes, files, records, and document descriptions, and tests whether the corresponding number in arkivuttrekk.xml (inneholderSkjermetInformasjon) is correct`